README.md

```diff
@@ -53,6 +53,30 @@ Some WordPress related configuration has to be done manually after the devstack
 6. Go to `Appearance -> Themes` and activate `Edly WordPress Theme`
 7. Enjoy.
 
+To setup the Edly edx themes repo.
+
+
+1. Go do the `edly/edX` directory.
+2. Clone [edly-edx-themes](https://github.com/edly-io/edly-edx-themes) in the `src` folder.
+3. Run `sudo chmod -R 777 src/`
```

Comment:
I think we can skip this step

Reply:
This is required to handle permission issues between docker containers and the host OS.